### PR TITLE
Add Mia to Design System team page

### PR DESCRIPTION
**Pull request by kellylee-gds**

### CODE REVIEW

File: src/design-system-team.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ If you want to contact the team you can [get in touch via email or Slack](/get-i
 - Kam Nijjar – Associate Delivery Manager
 - Katrina Birch – User Researcher
 - Kim 'beeps' Grey – Frontend Developer
+- Mia Allers – Graphic Designer
```
Comment (owenatgov) [inline src/design-system-team.md:32]: I'm not sure how much this matters but I can't remember if Mia is a senior or lead graphic designer. It's not exactly a show stopper but other folks have senior in their title on this page.

**Review by owenatgov — APPROVED** [review]
Tiny comment which I'm not sure is worth futzing over. Approving anyway.

File: src/design-system-team.md (hunk 1 above)
Comment (kellylee-gds) [reply src/design-system-team.md:32]: Good spot! She is senior...will amend 👍🏻

**Review by owenatgov — APPROVED** [review]
Double approval!

**Pull request merged**